script/Arcueid/Day 13/13_00_ARC13_1B.txt

```diff
@@ -1049,17 +1113,20 @@ If it's about Roa, he will be taken care of in a couple of days."
 [sha:18b80606a95832077d7a6e7adfd3025076bcf61b]{
 -- Page 164, Offset 32114.
 -- 「ロアの転生体が特定できましたから、法王庁……えっと、わたしたちの本拠地ですけど、そこに要請が通ります。
-"Since we've identified Roa's reincarnate, we'll finally be able to request aid from the Curia... uhhh, our headquarters.
+"Now that we've identified Roa's reincarnation, we can finally request assistance from the Vatican... uhh, our headquarters.
```

Comment:
I'm not catholic so I'm going off of Wikipedia descriptions here, but the Curia appears to be a specifically the authoritative body in the church. I'm guessing you're using "Vatican" here like "White House" or "Pharaoh" but even so, I think that Curia is right.
> comprises the administrative institutions of the [Holy See](https://en.wikipedia.org/wiki/Holy_See)[[note 1]](https://en.wikipedia.org/wiki/Roman_Curia#cite_note-1) and the central body through which the affairs of the [Catholic Church](https://en.wikipedia.org/wiki/Catholic_Church) are conducted. It acts in the [pope](https://en.wikipedia.org/wiki/Pope)'s name and with his authority for the good and for the service of the [particular churches](https://en.wikipedia.org/wiki/Particular_church) and provides the central organization for the church to advance its objectives.[[1]](https://en.wikipedia.org/wiki/Roman_Curia#cite_note-2)[[2]](https://en.wikipedia.org/wiki/Roman_Curia#cite_note-3)[[3]](https://en.wikipedia.org/wiki/Roman_Curia#cite_note-4)[[note 2]](https://en.wikipedia.org/wiki/Roman_Curia#cite_note-6)